src/vitabel/timeseries.py

```diff
@@ -1728,6 +1728,9 @@ def __init__(
         annotation_preset_type: LabelAnnotationPresetType | None = None,
     ):
 
+        if time_index is not None and len(time_index) > 0 and all(isinstance(item, tuple) for item in time_index):
+            time_index = [interval for time_tuple in time_index for interval in time_tuple]
+        
```

Comment:
This is a pretty good idea, I'd be a bit more strict about the interval tuples though. What do you think about this:

```suggestion
        if time_index is not None and len(time_index) > 0:
            time_index = np.array(time_index)
            if time_index.ndim == 2 and time_index.shape[1] == 2:
                # time data passed as pairs of interval end points
                time_index = time_index.reshape(-1)
```

in all other cases, i'd leave the input data as is.

Reply:
@behackl can you help with the docstring?